src/roomba_robot/tasks/botgal_and_lib/main.c

```diff
@@ -51,7 +51,7 @@ bool setupcamera() {
 void setuprobot(bool debug, bool camera) {
     create_connect();
     setupsensordata();
-    if (debug == true) {
+    if (debug) {
```

Comment:
Not quite with c preprocessors.... I'll just take a look at it myself

Reply:
So what I mean by this is something along this line:

```c
#define DEBUG true // at the top of the "true" main.c file.

// ...


#ifdef DEBUG
    printsensordata(); // will only be in the code if DEBUG is defined, so if we remove the #define DEBUG statement, this code will also vanish; if you keep the if-statement in the code like it is currently, it will always be checked if debug is true or not, regardless of circumstances.
#endif

```